pkg/azuredisk/nodeserver.go

```diff
@@ -351,3 +340,82 @@ func getFStype(attributes map[string]string) string {
 
 	return ""
 }
+
+func (d *Driver) checkMountPoint(target string) error {
```

Comment:
what does this function do? any comments? actually it does not only do `checkMountPoint`, also makes dir.

Reply:
done.